spec/rspec/core/reporter_spec.rb

```diff
@@ -36,6 +36,10 @@ module RSpec::Core
         reporter.finish
       end
 
+      it "will allow profiling when activated after formatter is used" do
+        config.profile_examples = 10
+        reporter.finish
+      end
```

Comment:
It's not clear to me how the this example shows a formatter being used before profiling is activated--profiling activation looks like the first thing that happens in this spec.

Reply:
Changed to stub the config check to make it clearer, as profiler activation was manual before this actually did break without the code change tho.